### DISCUSSION TITLE
Grant money

### DISCUSSION THREAD

**Discussion by misciel** [microCOVID/microCOVID#1786] [General]
### Discussed in https://github.com/microCOVID/microCOVID/discussions/1626

I’m a historian of medicine, and I echo the sentiments that microcovid is an amazing tool that should be maintained. I use it regularly. I would gladly volunteer weekly time to help maintain it!! But I don’t know the stats or how to code… :-(

Has looking into grant money been considered? What about trying to reach out to Fauci? I wouldn’t be surprised if he too values this tool and could provide valuable feedback for its sustenance, not to mention - in a best case scenario - being willing to lend his name and his connections to advance it. 

Please let me know whatever I can do to help!

**Comment by apiology** [comment]
@misciel: That sounds interesting, and it's definitely crossed my mind, speaking for myself.  Would you be interested in taking point on this process?  I do not know much about this area, but would be happy to jump on a Zoom call with someone who knows more to brainstorm next steps and set up a division of responsibilities to get them what they need to make progress.

**Comment by misciel** [comment]
Sure! I can even reach out to the grant person at my college, if you’d like!Dictated from my iPhoneOn Feb 5, 2023, at 1:24 PM, Vince Broz ***@***.***> wrote:﻿
@misciel: That sounds interesting, and it's definitely crossed my mind, speaking for myself.  Would you be interested in taking point on this process?  I do not know much about this area, but would be happy to jump on a Zoom call with someone who knows more to brainstorm next steps and set up a division of responsibilities to get them what they need to make progress.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

Saint Joseph's College of Maine278 Whites Bridge Road | Standish, ME 04084  sjcme.edu

**Reply by apiology** [reply]
That'd be great, @misciel!  Would it work to talk sometime soon, even if you haven't had a chance to reach out to that person yet?  You can [book some time here](https://calendly.com/apiology/microcovid) if that's convenient.  I can give you my idea of what I think might and might not be workable from the project's side.  I'd love to get your best understanding at this point of what to expect.

(Same goes for anyone else out there who may know about these things - the more the merrier!)

**Comment by misciel** [comment]
Still waiting to hear from our grants person… I reached out to some of our science faculty because much of this work would be extremely meaningful project-based learning for students, but they are already maxed out on their own projects. However, we could consider reaching out to faculty at other institutions to see if they could host the project there and integrate it into their teaching & research programs. Otherwise, I think it would be best to organize microCOVID as a nonprofit, which would facilitate fundraising, among other things.I’ll check with our grants person again. If I would be the primary investigator on a grant, probably the best I could muster, based on my expertise, would be something in the digital or medical humanities, but that’s not where the bigger bucks are.If my grants person doesn’t reply or isn’t interested, I’ll try to organize a meet for just me towards the end of this week…Talk to you soon!Michelle On Feb 8, 2023, at 6:26 AM, Vince Broz ***@***.***> wrote:﻿
That'd be great, @misciel!  Would it work to talk sometime soon, even if you haven't had a chance to reach out to that person yet?  You can book some time here if that's convenient.  I can give you my idea of what I think might and might not be workable from the project's side.  I'd love to get your best understanding at this point of what to expect.
(Same goes for anyone else out there who may know about these things - the more the merrier!)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

Saint Joseph's College of Maine278 Whites Bridge Road | Standish, ME 04084  sjcme.edu

**Comment by apiology** [comment]
@misciel: Sounds good - I look forward to talking and learning about this process!  Feel free to throw me some times if there's nothing convenient on the link above.
